crates/swc_ecma_transforms_base/src/fixer.rs

```diff
@@ -577,6 +577,18 @@ impl VisitMut for Fixer<'_> {
 
             _ => {}
         }
+
+        dbg!(&*n.obj);
```

Comment:
cc @Austaras I think AST definition is wrong.
We can't store `ParenExpr` in `OptChainBase`. Any idea?
Should we add a variant to it?

Reply:
Maybe it should be `Paren (OptChain (OptChainMember))`?

Reply:
It does not work.

```js
global.a = null;
let foo = "PASS";
a?.b[(foo = "FAIL")];
console.log(foo);
```

this is a terser test case, and it should **not** be


```js
global.a = null;
let foo = "PASS";
(a?.b)[(foo = "FAIL")];
console.log(foo);
```

Reply:
If `a?.b[(foo = "FAIL")];` is the raw input, it should be `OptChain(OptChainMember (Member))` while `(a?.b)[(foo = "FAIL")]` should be `Member(OptChain(OptChainMember))`, at least this is what babel is doing

Reply:
Thanks! I'm not sure if it's ideal, but we can avoid a breaking change for wasm plugin authors with that way 👍